src/code/fv-components/fv-change-sex-toggle.js

```diff
@@ -0,0 +1,50 @@
+import React, {PropTypes} from 'react';
+import classNames from 'classnames';
+import t from '../utilities/translate';
+
+/**
+ * Stateless functional React component for displaying male/female change buttons
+ * The appearance of the buttons is currently entirely controlled via external CSS.
+ * @param {integer} sex - [BioLogica.MALE, BioLogica.FEMALE] currently selected button
+ * @param {function} onChange(evt, sex) - callback to be called when use clicks to change sex
+ */
+const FVChangeSexToggle = ({sex, style={}, onChange}) => {
+  const isMaleSelected = sex === BioLogica.MALE,
+        isFemaleSelected = sex === BioLogica.FEMALE,
```

Comment:
Makes sense! I'm all for less male-centric variables, and since they're `const` I guess there's no danger of having them get out of sync